lib/oli_web/live/curriculum/container/container_live.ex

```diff
@@ -86,21 +89,28 @@ defmodule OliWeb.Curriculum.ContainerLive do
            children: children,
            active: :curriculum,
            breadcrumbs:
-             Breadcrumb.trail_to(project_slug, container.slug, Oli.Publishing.AuthoringResolver, project.customizations),
+             Breadcrumb.trail_to(
+               project_slug,
+               container.slug,
+               Oli.Publishing.AuthoringResolver,
+               project.customizations
+             ),
```

Comment:
This here was done by the formatter maybe? 😅

Reply:
Yes, I ran `mix format` in all the files I changed